Tests/Symmetry/DynamicTest/Model/MuscleSymmetryCheck.any

```diff
@@ -157,19 +171,24 @@ AnyFolder SymmetryCheck= {
 
           AnyKinCoM Thigh ={   AnySeg  &ref= Main.HumanModel.BodyModel.Left.Leg.Seg.Thigh;      };
           AnyKinCoM Shank={   AnySeg  &ref= Main.HumanModel.BodyModel.Left.Leg.Seg.Shank;      };
-          AnyKinCoM Talus={   AnySeg  &ref= Main.HumanModel.BodyModel.Left.Leg.Seg.Talus;      };
+        #if BM_LEG_MODEL != _LEG_MODEL_LEG_
+            AnyKinCoM Talus={   AnySeg  &ref= Main.HumanModel.BodyModel.Left.Leg.Seg.Talus;      };
+        #endif
           AnyKinCoM Foot={   AnySeg  &ref= Main.HumanModel.BodyModel.Left.Leg.Seg.Foot;      };
         };
 
         AnyFolder Comparison ={
           AnyFloat CoMDif =(.Right.CoMAll.Pos-.Left.CoMAll.Pos)* {{1,0,0},{0,1,0},{0,0,0}};
           AnyFloat Thigh =(.Right.Thigh.Pos-.Left.Thigh.Pos)* {{1,0,0},{0,1,0},{0,0,0}};
           AnyFloat Shank=(.Right.Shank.Pos-.Left.Shank.Pos)* {{1,0,0},{0,1,0},{0,0,0}};
+        #if BM_LEG_MODEL != _LEG_MODEL_LEG_
           AnyFloat Talus=(.Right.Talus.Pos-.Left.Talus.Pos)* {{1,0,0},{0,1,0},{0,0,0}};
+        #endif
           AnyFloat Foot=(.Right.Foot.Pos-.Left.Foot.Pos)* {{1,0,0},{0,1,0},{0,0,0}};
         };
       };
-      #if BM_LEG_MUSCLES_BOTH == OFF
+      #if  ((BM_LEG_MUSCLES_RIGHT != OFF) & (BM_LEG_MUSCLES_LEFT != OFF))

```

Comment:
Why do you check for the leg muscles here? Should it not be the arm muscles. I guess it should have been: 
```suggestion
      #if  ((BM_ARM_MUSCLES_RIGHT == OFF) & (BM_ARM_MUSCLES_LEFT == OFF))
```
I think there is bug, where AnyBody crashes, when the class template `ComparePtrArrays` gets finds an empty array. But fixing the BM check should prevent it from including that code unless there are joint-muscles in the model.